dask/dataframe/core.py

```diff
@@ -3943,13 +3952,16 @@ def _rename_dask(df, names):
     return new_dd_object(graph, name, metadata, df.divisions)
 
 
-def quantile(df, q):
+def quantile(df, q, method='default'):
     """Approximate quantiles of Series.
 
     Parameters
     ----------
     q : list/array of floats
         Iterable of numbers ranging from 0 to 100 for the desired quantiles
+
+    Note: this implementation will use t-digest is `method` is set to `tdigest` and the
+          dtype of df is float. Otherwise it falls back to the internal implementation.
```

Comment:
Should put this as a parameter docstring:

```python
"""
...
method : {'default', 'tdigest', 'dask'}:
    What method to use. By default will use dask's internal custom algorithm (``'dask'``).
    If set to ``'tdigest'`` will use tdigest for floats and ints and fallback to the ``'dask'``
    otherwise.
...
"""
```